app/assets/stylesheets/collection-items.scss

```diff
@@ -0,0 +1,8 @@
+$selected-background: navy;
```

Comment:
styles used to highlight selected items in list.
I've used SCSS, so you can change those two constants only and they'll be used in underlying styles